nextbus_setup/buildAggregatedData.js

```diff
@@ -22,7 +22,7 @@ const getRouteTags = directory => {
 const addNewStops = stops => {
   let duplicateCounter = 0;
   stops.map(stop => {
-    if (!stopPoints[stop.stopId]) stopPoints[stop.stopId] = stop;
+    if (!stopPoints[stop.tag]) stopPoints[stop.tag] = stop;
```

Comment:
this will break the code unless I update the relevant fields elsewhere. Needs to be done though.

stopId is a weird alternative tag option that isn't used throughout their API so I need to swap over to tags.